site/_guides/grpc-tls-howto.md

```diff
@@ -112,4 +112,4 @@ Note that we don't put the CA **key** into the cluster, there's no reason for th
 # Conclusion
 
 Once this process is done, the certificates will be present as Secrets in the `projectcontour` namespace, as required by
-[examples/contour]({{site.github.repository_url}}/tree/master/examples/contour).
+[examples/contour]({{site.github.repository_url}}/blob/{{ site.github.latest_release.tag_name }}/examples/contour).
```

Comment:
`https://github.com/projectcontour/contour/blob/v0.15.3/examples/contour` is a 404